### PR TITLE
RT1.3 updated route-policy under neighbor afisafi

### PR DESCRIPTION
**Pull request by singh-prem**
Uplifted RT1.3 with
1.  route-policy configuration under neighbor afisafi
2. afisafi under neighbor

### CODE REVIEW

**Comment by singh-prem** [comment]
logs
`
--- PASS: TestBGP (245.25s)
    --- PASS: TestBGP/propagate_IPv4_over_IPv4 (110.89s)
    --- PASS: TestBGP/propagate_IPv6_over_IPv6 (134.35s)
    --- SKIP: TestBGP/propagate_IPv4_over_IPv6 (0.00s)
PASS

*** Releasing the testbed...

*** PROPERTY: time.begin -> 1674193535
*** PROPERTY: time.end -> 1674193928
ok  	github.com/openconfig/featureprofiles/feature/bgp/addpath/ate_tests/route_propagation_test	
`

File: feature/experimental/p4rt/otg_tests/traceroute_packetout_test/rundata_test.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Code generated by go run tools/addrundata; DO NOT EDIT.
+package traceroute_packetout_test
+
+import "github.com/openconfig/featureprofiles/internal/rundata"
+
+func init() {
```
Comment (prinikasn) [inline feature/experimental/p4rt/otg_tests/traceroute_packetout_test/rundata_test.go:6]: Do we need this file in this PR ?

**Review by prinikasn — COMMENTED** [review]
/gcbrun

File: feature/experimental/p4rt/otg_tests/traceroute_packetout_test/rundata_test.go (hunk 1 above)
Comment (prinikasn) [reply feature/experimental/p4rt/otg_tests/traceroute_packetout_test/rundata_test.go:6]: Checked with the team. Looks like this test was added in PR908, but the rundata file was not included in that PR. and w/o this rundata_test file, this PR is failing the checks required to merge.

**Review by prinikasn — APPROVED** [review]

**Review by prinikasn — APPROVED** [review]

**Review by thesrinath — APPROVED** [review]

**Review by jasdeep-hundal — APPROVED** [review]

**Pull request merged**